invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/search/components.js

```diff
@@ -29,15 +29,20 @@ export const RDMRecordResultsListItem = ({ result, index }) => {
   );
   const publicationDate = _get(
     result,
-    "ui.publication_date_l10n",
+    "metadata.publication_date_l10n",
     "No publication date found."
   );
+  const updatedDate = _get(
+    result,
+    "updated_date_l10n",
```

Comment:
Minor: It should live inside ``ui`` key. If it doesnt' there's something wrong in the REST API seriazlier.

```
{
  "ui": {
    "publication_date_l10n": "...",
    "updated_l10n": "...",
  }
}

Reply:
See also nveniosoftware/invenio-rdm-records#203